0-ad-history-mod-package/simulation/templates/structures/athen/house.xml

```diff
@@ -4,11 +4,10 @@
     <Civ>athen</Civ>
     <SpecificName>Oikos</SpecificName>
     <History>
-    Unfortunately, houses in Ancient Greece have not aged well, and usually the most archeologists can find is a foundation. While renown for their magnificent stone temples, the Greeks apparently skimped on their own houses. A house typically was constructed from mud bricks. Which wasn’t durable at all, so much that burglars usually dug a hole in the wall rather than breaking through the wall.
-    Evidence shows that Athenian houses frequently had a second story. And sometimes a sort of tower, however this feature was more likely in the countryside. Inside, the house the walls were usually whitewashed, while the floor was usually just packed earth or paving stones. The houses had windows except they were tiny but built very close to the ceiling.
-    Greek household or oikos, usually held extended family with grandparents, parents, sons, their wives, and children, unmarried daughters, and slaves if they were wealthy enough to have them. However, only the most trusted slaves would live in the main house, the rest would be quartered in a separate building, possibly with the animals.
-    The houses of the middle and upper class were built around a central courtyard that led off to the different rooms such as storerooms, the andrôn where the man hosted his symposium, and the gynaikôn or women’s quarters. It is debated whether the gynaikôn was where the women lived, or whether it was where they worked and spent their free time. However, is some literary evidence to support the former.
-
+Unfortunately, houses in Ancient Greece have not aged well, and usually the most archeologists can find is a foundation. While renown for their magnificent stone temples, the Greeks apparently skimped on their own houses. A house typically was constructed from mud bricks. Which wasn’t durable at all, so much that burglars usually dug a hole in the wall rather than breaking through the wall.
```

Comment:
```suggestion
Unfortunately, houses in Ancient Greece have not aged well, and usually the most archeologists can find is a foundation. While renown for their magnificent stone temples, the Greeks apparently skimped on their own houses. A house typically was constructed from mud bricks which wasn’t durable at all. So much so, in fact, that burglars usually dug a hole in the wall rather than breaking through it.
```